src/firebase/generic-db.js

```diff
@@ -12,7 +13,7 @@ export default class GenericDB {
    * @param id
    */
   async create(data, id = null) {
-    const collectionRef = firebase.firestore().collection(this.collectionPath)
+    const collectionRef = (await firestore()).collection(this.collectionPath)
```

Comment:
I'm not a big fan of this `(await firestore())` statement everywhere, but I'm sure we can improve this

Reply:
Hmmm, I made so research, and couldn't find a better syntax than yours. But after all I think it's ok because firestore is not supposed to be imported in other files than here

Reply:
I agree we that. Ugly, but it is acceptable.